diff --git a/share/applications/torbrowser-settings.desktop b/share/applications/torbrowser-settings.desktop
index 4f12adb..867bb5a 100644
--- a/share/applications/torbrowser-settings.desktop
+++ b/share/applications/torbrowser-settings.desktop
@@ -1,5 +1,6 @@
 [Desktop Entry]
 Name=Tor Browser Launcher Settings
+Name[cs]=Tor Browser Launcher nastavení
 Name[fr]=Tor Browser Launcher configurations
 Name[hu]=Tor-böngésző indító beállításai
 Name[nl]=Tor Browser Launcher Instellingen
@@ -8,6 +9,7 @@ Name[ru]=Tor Browser (настройки запуска)
 GenericName=Tor Browser Launcher Settings
 GenericName[hu]=Tor-böngésző indító beállításai
 Comment=Tor Browser Launcher Settings
+Comment[cs]=Tor Browser Launcher nastavení
 Comment[fr]=Tor Browser Launcher configurations
 Comment[hu]=Tor-böngésző indító beállításai
 Comment[nl]=Tor Browser Launcher Instellingen
diff --git a/share/applications/torbrowser.desktop b/share/applications/torbrowser.desktop
index 7a25076..e5f9cd0 100644
--- a/share/applications/torbrowser.desktop
+++ b/share/applications/torbrowser.desktop
@@ -1,10 +1,12 @@
 [Desktop Entry]
 Name=Tor Browser
+Name[cs]=Tor Browser
 Name[hu]=Tor-böngésző
 Name[pt_BR]=Navegador Tor
-GenericName=Tor browser
+GenericName=Tor Browser
 GenericName[hu]=Tor böngésző indító
 Comment=Launch Tor Browser
+Comment[cs]=Spustit Tor Browser
 Comment[hu]=Tor böngésző indító
 Comment[pt_BR]=Navegador Tor
 Exec=torbrowser-launcher %u
