diff --git a/torbrowser_launcher/common.py b/torbrowser_launcher/common.py
index 1a05fb4..913485c 100644
--- a/torbrowser_launcher/common.py
+++ b/torbrowser_launcher/common.py
@@ -255,7 +255,7 @@ class Common(object):
         # Fetch key from wkd, as per https://support.torproject.org/tbb/how-to-verify-signature/
         p = subprocess.Popen(
             [
-                "gpg2",
+                "gpg",
                 "--status-fd",
                 "2",
                 "--homedir",
